src/pages/Components/RatingStars/code.md

```diff
@@ -3,39 +3,198 @@ title: 'Code'
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
+  </label>
+</div>
 ```
 
-<preview path="src/pages/Components/RatingStars/previews/RatingStars"></preview>
+<preview path="src/pages/Components/RatingStars/previews/stars-input-basic"></preview>
+
+### Variations
+
+#### Available sizes
 
-## Variations
+The rating stars are available in 4 sizes.
 
-use the same variations in the same order as the index page
+To change the size of the stars, simply apply one of the following modifier **to the main container** :
 
-## Behaviors
+- **small** : `mc-stars-input--s`
+- **medium** : this is the default size so you don't need to add a modifier class
+- **large** : `mc-stars-input--l`;
+- **extralarge** : `mc-stars-input--xl`;
+
+```html
+<div class="mc-stars-input mc-stars-input--s">...</div>
+
+<div class="mc-stars-input mc-stars-input--l">...</div>
+```
+
+<preview path="src/pages/Components/RatingStars/previews/stars-input-all-sizes"></preview>
+
+### Behaviors
 
 use the same Behaviors in the same order as the index page
 
-## Responsive behaviors
+### Responsive behaviors
 
 use the same Behaviors in the same order as the index page
 
-## Extension and customization
+### Extension and customization
 
 how to use the mixins and function to customize
 
-## accessibility and semantic
+### accessibility and semantic
+
+<hintitem>
+  Each radio button <strong>(input.mc-stars-input__radio)</strong> must be associated with a label tag <strong>(label.mc-stars-input__label)</strong>
+</hintitem>
+
+<hintitem>
+  Each label <strong>(label.mc-stars-input__label)</strong> must contain an explicit text describing the value of the radio button
+</hintitem>
+
+<hintitem>
+  The label text must be wrapped with a <strong>span.sr-only.mc-stars-input__text tag</strong>. The <strong>sr-only</strong> class, allows to visually hide the text but to keep it accessible to screen readers
+</hintitem>
+
+<hintitem dont=true>
+  Don't forget to insert the first radio button that has a neutral value. This radio button must <strong>NOT</strong> be associated with a label tag. However, in this case the label is replaced by an <strong>aria-label</strong> attribute.
+</hintitem>
+
+## When displaying the global score
+
+<a name="case-2"></a>
+
+### Import
+
+Import the settings and the stars `scss` files
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.stars-result';
+```
+
+### Basic usage
+
+To use rating stars when displaying the global score on a product sheet, you must apply the `mc-stars-result` class to a `div` serving as a container for `span.mc-stars-result__icon` tags.
+
+```html
+<div class="mc-stars-result">
+  <span class="mc-stars-result__icon"></span>
+  <span class="mc-stars-result__icon"></span>
+  <span class="mc-stars-result__icon"></span>
+  <span class="mc-stars-result__icon"></span>
+  <span class="mc-stars-result__icon"></span>
+  <span class="sr-only mc-stars-result__text">Global score: 0</span>
```

Comment:
maybe we can achieve that with an aria-label or a title or other stuff

Reply:
and I'm sure that there is attribute to help robots to know that this is a notation